DisguiseUnityRenderStream/Editor/DisguiseRenderStreamBuild.cs

```diff
@@ -172,5 +177,54 @@ static void AddSceneToSchema(ManagedSchema schema, int sceneIndex, string name)
             
             currentScene.parameters = parameters.ToArray();
         }
+        
+        static void CheckVsync()
+        {
+            if (DisguiseFramerateManager.Enabled && DisguiseFramerateManager.VSyncIsEnabled)
+            {
+                Debug.LogWarning($"DisguiseRenderStream: {nameof(QualitySettings)}{nameof(QualitySettings.vSyncCount)} is currently enabled. For best performance disable vSync in the project settings.");
```

Comment:
But when testing to validate my understanding of the nameof (would it add a . automatically?) I faced a few things:
1. In my setup DisguiseFramerateManager.Enabled was always false, even if I have the cluster display package in the project.  So it looks like ENABLE_CLUSTER_DISPLAY is not defined.  What step have I missed?
2. Regardless, shouldn't we do that check all the time, even without cluster display data replication?
3. I must have missed yet another thing, but even if I remove the if I was not able to see the warning in the console.

Reply:
1. Ah good catch, the #define is coming from RenderStream.Disguise.asmdef. The editor assembly needs the same versionDefine, adding. So with the fix: DisguiseFramerateManager is enabled when ClusterDisplay is not in the project.
![image](https://user-images.githubusercontent.com/84032434/227571908-2f038e47-c529-46f1-9992-cfa595fdd877.png)

2. I prefer giving ClusterDisplay full control over the framerate settings when it's in the project to avoid future conflicts. If vsync is a factor in ClusterDisplay we should add a similar check to it.

3. I couldn't repro. I added a `DisguiseFramerateManager.Reset()` method though, maybe the flag was interfering?

Reply:
We discovered "Clear on Recompile" will clear all console messages from OnPreprocessBuild.
![image](https://user-images.githubusercontent.com/84032434/227612271-ad8382f8-1a08-4ce2-a389-8255f748a2dc.png)

Reply:
The updated approach is:
* Only throw exceptions in pre-build
* Defer logs to post-build
* Automatically disable vSync in the project in pre-build, with warning logged in post-build
* The frame manager (ensures vSync off + unlimited Application.targetFrameRate) runs even with the ClusterDisplay package, since Disguise is still the output and not the local display